app/models/post.rb

```diff
@@ -3,6 +3,7 @@ class Post < ActiveRecord::Base
   # Associations
   belongs_to :council
   has_and_belongs_to_many :profiles
+	has_and_belongs_to_many :elections
```

Comment:
Inconsistent indentation detected.<br>Tab detected.<br>Prefer `has_many :through` to `has_and_belongs_to_many`.